aws/rds_enhanced_monitoring/README.md

```diff
@@ -133,14 +133,35 @@ Process a RDS enhanced monitoring DATA_MESSAGE, coming from CLOUDWATCH LOGS
 
 # Setup
 
-1. Create a KMS key for the Datadog API key and app key
-   - Create a KMS key. Refer to the [AWS KMS Creating Keys][1] documentation for step by step instructions.
-   - Encrypt the token using the AWS CLI. `aws kms encrypt --key-id alias/<KMS key name> --plaintext '{"api_key":"<dd_api_key>", "app_key":"<dd_app_key>"}'`
-   - Make sure to save the base-64 encoded, encrypted key (`CiphertextBlob`). This is used for the `KMS_ENCRYPTED_KEYS` variable in all lambda functions.
-   - Optional: set the environment variable `DD_SITE` to `datadoghq.eu` to automatically forward data to your EU platform.
```

Comment:
I think we should keep something in the doc about using this env variable if in eu.

Reply:
Whoops, adding that back